src/core/lv_obj_scroll.c

```diff
@@ -309,7 +309,7 @@ void lv_obj_scroll_by(lv_obj_t * obj, int32_t dx, int32_t dy, lv_anim_enable_t a
         lv_anim_t a;
         lv_anim_init(&a);
         lv_anim_set_var(&a, obj);
-        lv_anim_set_completed_cb(&a, scroll_completed_completed_cb);
```

Comment:
Why was it modified? it comes from this fix: https://github.com/lvgl/lvgl/pull/5979

Reply:
I reverted it.